lib/modules/manager/dockerfile/extract.ts

```diff
@@ -283,6 +283,25 @@ export function extractPackageFile(
       instruction += '\n' + lineLookahead;
     }
 
+    const syntaxRegex = regEx('^#[ \\t]*syntax=(?<image>\\S+)', 'im');
```

Comment:
why case insensitive? is this allowed too:
```Dockerfile
# Syntax=docker/dockerfile:1

FROM scratch
```

Reply:
@SierraNL, why did you remove the case-insensitive flag now? The sample @viceice posted is valid too too and should be supported as well.

Reply:
Ok, I thought it wasn't possible, I could not find examples with uppercase.